### PR TITLE
NN-3799: Add createdUserId to reported adjudications

### PR DESCRIPTION
**Pull request by sp-watson**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/dtos/ReportedAdjudicationDto.kt [hunk 1]
```diff
@@ -17,5 +17,7 @@ data class ReportedAdjudicationDto(
   @ApiModelProperty(value = "Incident details")
   val incidentDetails: IncidentDetailsDto,
   @ApiModelProperty(value = "Incident statement")
-  val incidentStatement: IncidentStatementDto
+  val incidentStatement: IncidentStatementDto,
```
Comment (sp-watson) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/dtos/ReportedAdjudicationDto.kt:20]: This is what is sent back to the API caller

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/gateways/ReportedAdjudication.kt [hunk 2]
```diff
@@ -13,7 +13,8 @@ data class ReportedAdjudication(
   val agencyId: String,
   val incidentTime: LocalDateTime,
   val incidentLocationId: Long,
-  val statement: String
+  val statement: String,
+  val createdByUserId: String
```
Comment (sp-watson) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/gateways/ReportedAdjudication.kt:17]: This is what is returned by Prison API

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/health/HealthCheckTest.kt [hunk 3]
```diff
@@ -101,4 +101,19 @@ class HealthCheckTest : IntegrationTestBase() {
         }
       )
   }
+
```
Comment (sp-watson) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/health/HealthCheckTest.kt:104]: I spotted this missing test this morning, and fixed it before i knew about this change - so I lumped it in with this PR.

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/IntTestData.kt [hunk 4]
```diff
@@ -46,7 +48,19 @@ class IntTestData(
       locationId = 12L,
       dateTimeOfIncidentISOString = "2020-12-14T09:00:00",
       dateTimeOfIncident = LocalDateTime.parse("2020-12-14T09:00:00"),
-      statement = "Different test statement"
+      statement = "Different test statement",
+      createdByUserId = "P_NESS"
+    )
+
```
Comment (sp-watson) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/IntTestData.kt:54]: Added some more test data to the existing int tests

**Review by richardpopple — APPROVED** [review]

**Pull request merged**